core/profiles/standard/standard.install

```diff
@@ -346,14 +346,17 @@ function standard_install() {
 
   // Enable default permissions for system roles.
   $filtered_html_permission = filter_permission_name($filtered_html_format);
-  user_role_grant_permissions(BACKDROP_ANONYMOUS_ROLE, array('access content', 'access comments', $filtered_html_permission));
-  user_role_grant_permissions(BACKDROP_AUTHENTICATED_ROLE, array('access content', 'access comments', 'post comments', 'skip comment approval', 'upload editor images', $filtered_html_permission));
+  user_role_grant_permissions(BACKDROP_ANONYMOUS_ROLE, array('access content', 
```

Comment:
We only break inline comment lines at 80 chars. If you want to shorten this, the use "helper" array variables, like `$anonymous_permissions = array('access content', 'access comments', $filtered_html_permission);` and `$authenticated_permissions = array(bla bla bla);`. You can then break these arrays like so:

```php
$anonymous_permissions = array(
  'access content', 
  'access comments', 
  $filtered_html_permission,
);

...

$authenticated_permissions = array(
  'access content', 
  'access comments', 
  'post comments',
  'skip comment approval', 
  'upload editor images',
  $filtered_html_permission,
);
```


...and then do this:

```suggestion
  user_role_grant_permissions(BACKDROP_ANONYMOUS_ROLE, $anon_permissions);

  ...
  user_role_grant_permissions(BACKDROP_AUTHENTICATED_ROLE, $authenticated_permissions);
```